backend/plugin/db/mysql/mysql.go

```diff
@@ -177,6 +182,90 @@ func (driver *Driver) Execute(ctx context.Context, statement string, _ bool, opt
 			return 0, err
 		}
 	}
+
+	if opts.IndividualSubmission && len(statement) <= chunkedSubmissionMaximumSize {
+		return driver.executeChunkedSubmission(ctx, conn, statement, opts)
+	}
+
+	return driver.executeBatchSubmission(ctx, conn, statement, opts)
+}
+
+func (driver *Driver) executeChunkedSubmission(ctx context.Context, conn *sql.Conn, statement string, opts db.ExecuteOptions) (int64, error) {
+	list, err := mysqlparser.SplitSQL(statement)
+	if err != nil {
+		return 0, errors.Wrapf(err, "failed to split sql")
+	}
+	if len(list) == 0 {
+		return 0, nil
+	}
+
+	tx, err := conn.BeginTx(ctx, nil)
+	if err != nil {
+		return 0, errors.Wrapf(err, "failed to begin execute transaction")
+	}
+	defer tx.Rollback()
+
+	// Round up the number of sql per chunk.
+	sqlPerChunk := (len(list)-1)/maxChunksCount + 1
```

Comment:
Is there a way that we can write a util splitting function with some unit testing?
1) util: other engines need it as well;
2) unit testing: I'm worried incorrect splitting will cause missing statements. "Hey, why I'm missing this data even though you've executed the task?"